AMW_angular/io/src/app/apps/app-add/app-add.component.html

```diff
@@ -0,0 +1,47 @@
+<div class="modal-header">
+  <h5 class="modal-title" id="editReleaseTitle">Create application</h5>
```

Comment:
Why is the id set to `editReleaseTitle` - is the id used in any name?

Reply:
Copy paste error